packages/cdk/utils/src/replace-units/properties.ts

```diff
@@ -0,0 +1,9 @@
+import { UnitsMap } from './types';
+
+export const DEFAULT_UNITS_MAP: UnitsMap = {
+    s: 1,
+    m: 60,
+    h: 3600,
+    d: 86400,
+    w: 604800
```

Comment:
add year?

Reply:
it is hard to add year
it can be 365 or 366 days depends on leap year or not

Reply:
@Schirbak what do you think, can we dynamic detect or throw exception about year?